v0.1/elements/electrodeContact.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/elements/electrodeContact.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SANDS definition of an electrode contact",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type", 
+    "@id",
+    "identifier",
+    "coordinate"
+    ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/ElectrodeContact"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier.",
+      "pattern": "sands/elements/electrodeContact/([A-Za-z0-9_-]+).json"
+    },
+    "identifier": {
+      "type": "string",
+      "description": "Identifier of the electrode contact within files."
+    },
+    "coordinate": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.schema.json#definitions/COORDINATE",
+      "description": "Central coordinate of the electrode contact."
+    },
+    "definedIn":{
+	  "type": "array",
+	  "minItems": 1,
+	  "items": {
+	    "oneOf": [
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json#definitions/CSCS_FILE"},
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE"}
+	    ],
+	  },
+      "description": "File in which the coordinate of this electrode contact were defined."
+    },
+    "relatedRecordings":{
+	  "type": "array",
+	  "minItems": 1,
+	  "items": {
+	    "oneOf": [
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json#definitions/CSCS_FILE"},
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE"}
+	    ],
+	  },
+      "description": "Files in which the recorded data from this electrode contact were stored."
+    },
+    "relatedStimulations":{
+	  "type": "array",
+	  "minItems": 1,
```

Comment:
Remove and add as required property.

Reply:
Ah no... I think you misunderstand the feature of *required* and *minItems*. 
Although you are correct that if you put an array property into *required*, the *minItems* become unnecessary (although not wrong), it does not work the other way around.  
*minItems* only means that if you provide a value for that property, you have to at least provide one entry. But if this property is not in required you do not even have to list it in the json later on.  
Thinking about it, I'm not sure though if in general the `"minItems": 1` gives any additional information. I think this one makes only sense if you would require at least 2 items or want to define a range (1-10).  
I remove the `minItems` for now, but we should make some research here considering good practice in writing json-schemas.

Reply:
Yeah, true. I got it like 80% correct. I forgot that this just means that when you use it, it has to have at least one item. Problem is taht this would be implicit anyway (given that when you want to use it, you cannot use less then one item...). 

I agree, with `"minItem": 2` it makes sense again to have it. 

Conclusion: Remove it either way. Thinking about it, it won't make sense to make this required of course (since it may not apply). So, don't add to `required`.